src/session.rs

```diff
@@ -334,9 +350,9 @@ impl Client {
     /// Issue the specified [`WebDriverCompatibleCommand`] to the WebDriver instance.
     pub async fn issue_cmd(
         &self,
-        cmd: impl WebDriverCompatibleCommand + Send + 'static,
+        cmd: Box<dyn WebDriverCompatibleCommand + Send + 'static>,
```

Comment:
How about taking
```suggestion
        cmd: impl Into<Box<impl WebDriverCompatibleCommand + Send + 'static>>,
```
and then below doing:
```rust
Cmd::WebDriver(Box::from(cmd) as Box<_>)
```
so that callers can also provide a non-boxed type for convenience?

Reply:
I agree, I’ve updated `issue_cmd` to use `impl Into<Box<impl WebDriverCompatibleCommand + Send + 'static>>`